.travis.yml

```diff
@@ -10,4 +10,4 @@ install:
   - travis_retry composer install --no-interaction
 
 script:
-  - vendor/bin/phpunit
+  - XDEBUG_MODE=coverage vendor/bin/phpunit --coverage-text
```

Comment:
There already is a code coverage test running in Scrutenizer. Do we need to do this a second time?

Reply:
Thanks for this pointer, scrapped the code coverage in #30 and that landed in this PR :+1: